### PR TITLE
Updated documentation of localization script related to use of individual GEN_OBS observations

### PR DESCRIPTION
**Pull request by oddvarlia**
Add documentation to updates related to use of individual observations from an GEN_OBS type of observation node in localisation.

### CODE REVIEW

File: tests/legacy_test_data/poly_gen_param/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+### Model update
+Then the loaded data is compared to the observed data and the parameters are
+updated accordingly.
+
+### Time
+Although we don't really have a clear concept of time in this case _ERT_
+expects us to have so. Since we have specified that all of our data is for time
+step `0`, we had to create an artificial time map `time_map` that specifies
+that the very first and only report step corresponds to `1/10/2006`. This could
```
Comment (berland) [inline tests/legacy_test_data/poly_gen_param/README.md:61]: Use ISO-8601 here, `2006-10-01`.

File: tests/legacy_test_data/poly_gen_param/time_map [hunk 2]
```diff
@@ -0,0 +1 @@
+1/10/2006
```
Comment (berland) [inline tests/legacy_test_data/poly_gen_param/time_map]: ISO-8601 here also.

Comment (oddvarlia) [reply tests/legacy_test_data/poly_gen_param/time_map]: OK

File: tests/legacy_test_data/poly_gen_param/README.md (hunk 1 above)
Comment (oddvarlia) [reply tests/legacy_test_data/poly_gen_param/README.md:61]: OK

**Comment by oddvarlia** [comment]
I have defined a commit containing the updated documentation in #363

**Pull request closed without merging**